Formula/passwdqc.rb

```diff
@@ -49,6 +48,9 @@ def install
       "SECUREDIR=#{prefix}/pam"
     end
 
+    # Workaround for Xcode 14.3
+    args << ("CFLAGS=-Wno-implicit-function-declaration" if DevelopmentTools.clang_build_version >= 1403)
```

Comment:
```suggestion
    # Workaround for Xcode 14.3
    ENV.append_to_cflags "-Wno-implicit-function-declaration" if DevelopmentTools.clang_build_version == 1403
    args << "CFLAGS=#{ENV.cflags}" if ENV.cflags.present?
```
Let's also report the implicit function declaration failure upstream.

Reply:
Upstream issue: https://github.com/openwall/passwdqc/issues/31